src/sauce-testreporter.ts

```diff
@@ -11,14 +11,19 @@ const getPlatformName = (platform: string) => {
   return platform;
 };
 
-export function generateJunitFile(
+export function generateJUnitFile(
   assetsPath: string,
   suiteName: string,
   browserName: string,
   platform: string,
 ) {
+  const junitPath = path.join(assetsPath, `report.xml`);
+  if (!fs.existsSync(junitPath)) {
+    console.warn(`JUnit file(${junitPath}) not found`);
```

Comment:
Shouldn't we indicate here to the user that we are not going to generate a junit file? You [changed the log output in the error scenario](https://github.com/saucelabs/sauce-testcafe-runner/pull/216/files#diff-8e0a3d9568b76eedb32e35049fef92c62507a612b611fcfb29e87ac1855fbec9R378), but not quite here. Just this log line alone won't be that useful to the user.